server-kotlin/src/city_sync/model/CityExport.kt

```diff
@@ -0,0 +1,157 @@
+package city_sync.model
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize
+import com.fasterxml.jackson.datatype.joda.deser.LocalDateDeserializer
+import org.jetbrains.exposed.sql.ResultRow
+import org.joda.time.LocalDate
```

Comment:
Why don't use Java `java.time.LocalDate`

Reply:
I had issue in converting between Exposed date type and java LocalDate, so I resolved it using joda LocalDate

Reply:
If you want I can help you with that (recently I did it for another project). But it can wait until you are ready, I do not want to slow you down.